src/main/java/frc/robot/RobotContainer.java

```diff
@@ -89,7 +92,7 @@ public RobotContainer() {
       e.printStackTrace();
     }
 
-    driveBaseSubsystem.setDefaultCommand(mDriveBaseTeleopCommand);
+    driveBaseSubsystem.setDefaultCommand(driveBaseTeleopCommand);
```

Comment:
Instead of putting it in Robot.java, I think you want to make a new InstantCommand for your expelBalls method and make it the default command for the intake subsystem.